src/app/component/Field/Field.component.js

```diff
@@ -148,16 +153,29 @@ class Field extends Component {
         );
     }
 
+    /**
+     * Render Type Text, default value is passed from parent
+     * handleToUpdate used to pass child data to parent
+     */
     renderTypeText() {
-        const { placeholder, id, isAutocompleteAllowed } = this.props;
+        const {
+            placeholder,
+            id,
+            isAutocompleteAllowed,
+            handleToUpdate,
+            originalValue
+        } = this.props;
         const { value } = this.state;
 
         return (
             <input
               type="text"
               id={ id }
-              value={ value }
-              onChange={ this.onChange }
+              defaultValue={ originalValue }
+              onChange={ (
+                  (typeof handleToUpdate === 'function') ? handleToUpdate({ id, value }) : null,
```

Comment:
Check whether handleToUpdate is a function should be performed in PropTypes, no need to check it here for the second time